File: app/src/main/java/com/lutireh/pettracker/domain/model/TaskType.kt
package com.lutireh.pettracker.domain.model

enum class TaskType(val label: String) {
    WALK("Passeio"),
    VACCINE("Vacina"),
    FEED("Alimentação"),
    BATH("Banho"),
    VET("Veterinário"),
    EXAM("Exame"),
    OTHER("Outro")
}

File: flutter_app/lib/domain/models/task_type.dart
enum TaskType {
  walk("Passeio"),
  vaccine("Vacina"),
  feed("Alimentacao"),
  bath("Banho"),
  vet("Veterinario"),
  exam("Exame"),
  other("Outro");

  const TaskType(this.label);
  final String label;

  static TaskType fromName(String value) {
    return TaskType.values.firstWhere(
      (type) => type.name == value,
      orElse: () => TaskType.other,
    );
  }
}
